bin/yunoprompt

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+ip=$(hostname --all-ip-address)
+
+i=0
+for key in /etc/ssh/ssh_host_*_key.pub ; do 
+    output=$(ssh-keygen -l -f $key)
+    fingerprint[$i]=" - $(echo $output | cut -d' ' -f2) $(echo $output| cut -d' ' -f4)"
+    i=$(($i + 1))
+done
+
+cat > /etc/issue.net << EOF
+ '.                        '                                    ''     -d.    
```

Comment:
We  should stay under the 80 char limit

Reply:
There is one line of 81 indeed and another of 80 the other make 79